inscriptions/ISic030011.xml

```diff
@@ -177,24 +183,30 @@
       <desc>Add Description Here</desc>
     </graphic>
   </surface>
-
     -->
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#TS"> <!-- add text, apparatus, translation and commentary -->
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#SN"> <!-- add text, apparatus, translation and commentary -->
                 <ab>
-                    <lb n="1"/>
-                    <lb n="2"/>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/><unclear>τ</unclear>ο͂ι Σο̄π<unclear>ά</unclear>τ<unclear>ρ</unclear>ο̄ι <space extent="unknown" unit="character"/>
+                    <lb n="2"/>ΓΥϙ<gap reason="lost" quantity="3" unit="character"/><unclear>ο</unclear>ς κ<unclear>α</unclear>ὶ ℎα Σο̄π<unclear>ά</unclear>τρο̄ γλο͂<supplied reason="lost"><choice><orig>σα</orig><reg>σσα</reg></choice></supplied>
+                    <lb n="3"/>Φρῦνις Νοαβ<unclear>αρ</unclear>ίλο̄ καὶ ℎα <supplied reason="lost">Φρύνιος <choice><orig>γ</orig></choice></supplied>
```

Comment:
for the lemmatisation which will be done later, it is necessary to add the <supplied> around each word. Same for the <choice><orig>/<reg> (but check if the validation fails with the line break):
`<supplied reason="lost">Φρύνιος</supplied> <choice><orig><supplied reason="lost">γ</supplied>
<lb n="4" break="no"/>λο͂σ<unclear>α</unclear></orig><reg><supplied reason="lost">γ</supplied>λο͂σσα</reg></choice> <surplus><choice><orig>γλο͂σα</orig><reg>γλο͂σσα</reg></choice></surplus>`